### PR TITLE
Can do Schematics select Edit such kind of operation after dead or while using unit can't build.

### PR DESCRIPTION
**Pull request by shugen002**
In some PvP server , player want to save schematics after dead or while observing competition. So i do some adjust for that.

### CODE REVIEW

File: core/src/mindustry/graphics/OverlayRenderer.java [hunk 1]
```diff
@@ -26,13 +26,15 @@
...
+        input.drawBottom();
+            
         if(player.dead()) return;
 
         if(player.isBuilder()){
             player.unit().drawBuildPlans();
         }
 
-        input.drawBottom();
+
```
Comment (joshuaptfan) [inline core/src/mindustry/graphics/OverlayRenderer.java:37]: Extraneous line break.

**Comment by shugen002** [comment]
Not all files in this project seems formated correctly as the file in .github/Mindustry-CodeStyle-IJ.xml . 
And I think we need someone to make a format check action for this project.

**Comment by Anuken** [comment]
The formatting config IntelliJ provides doesn't handle all the cases. Sometimes, I like to split statements into multiple indented lines, or (in very specific situations) omit spaces around operators.

**Comment by shugen002** [comment]
> The formatting config IntelliJ provides doesn't handle all the cases. Sometimes, I like to split statements into multiple indented lines, or (in very specific situations) omit spaces around operators.

i agree that. but at least it can help us avoid some changes just some spaces add and have a nice code history.

**Comment by shugen002** [comment]
Is there any other problem in this PR except the format action request?
The format action request can be discuss later in the issue or discord.

**Comment by shugen002** [comment]
close due to too long time.

**Comment by Anuken** [comment]
The issue with letting dead players poll input is that it has all sorts of potential side effects - the player may try to unload items or queue blocks to a non-existent unit.

**Pull request closed without merging**